python-sdk/src/astro/databases/snowflake.py

```diff
@@ -393,12 +392,12 @@ def create_stage(
         stage.set_url_from_file(file)
 
         fileformat = ASTRO_SDK_TO_SNOWFLAKE_FILE_FORMAT_MAP[file.type.name]
-        copy_options = [COPY_OPTIONS[file.type.name]]
+        copy_options = [COPY_OPTIONS.get(file.type.name)] if COPY_OPTIONS.get(file.type.name) else []
         copy_options.extend([f"{k}={v}" for k, v in self.load_options.copy_options.items()])
         file_options = [f"{k}={v}" for k, v in self.load_options.file_options.items()]
         file_options.extend([f"TYPE={fileformat}", "TRIM_SPACE=TRUE"])
         file_options_str = ", ".join(file_options)
-        copy_options_str = ", ".join(copy_options)
+        copy_options_str = ", ".join(copy_options)  # type:ignore
```

Comment:
what was the issue here?

Reply:
Following is the mypy error
```
Argument 1 to "join" of "str" has incompatible type "List[Optional[str]]"; expected "Iterable[str]"
```

Reply:
@sunank200 would it make sense to represent this argument as `Iterable[str]`?